features/il_display_assets_table/lib/src/widgets/assets_table_widget.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:il_core/il_entities.dart';
+
+class AssetsTableWidget extends StatelessWidget {
+  final List<Asset> assets;
+
+  const AssetsTableWidget({
+    super.key,
+    required this.assets,
+  });
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Consider optimizing the build method for large lists of assets to avoid performance issues.